### ISSUE TITLE
Unable to fetch logs

### ISSUE THREAD

**Issue by tume** [serverless/serverless-websockets-plugin#16]
I have following function in my serverless.yml 

```
functions:
  connections:
    handler: handler.connections
    events:
      - websocket:
          routeKey: $connect
      - websocket:
          routeKey: $disconnect
```

But can't fetch logs with ```sls logs -f connections```

Gives error:
```
  Serverless Error ---------------------------------------
 
  1 validation error detected: Value '[]' at 'logStreamNames' failed to satisfy constraint: Member must have length greater than or equal to 1

```

Using serverless 1.35.1.  Should the logs even work with websocket functions or should I maybe call it somehow differently?

**Comment by zhammer** [comment]
Just a heads up, this seems like a repeat of #13 (also unresolved)

**Comment by shortjared** [comment]
This should be fixed by #21, @tume please reopen if this is not the case for you!
